pyCGM_Single/pyCGM.py

```diff
@@ -954,25 +952,24 @@ def headJC(frame,vsk=None):
     Returns
```

Comment:
not part of this review, but I just noticed the vsk also says optional, which is not true.  The default is assigned as None, which will fail when a key is queried for headoffset.